From 155d304e9b5081d41b27dafa7a8ff0a7b6ce000c Mon Sep 17 00:00:00 2001
From: Richard Laager <rlaager@wiktel.com>
Date: Tue, 16 Jul 2024 15:30:50 +0800
Subject: [PATCH 2/5] Enable zed emails

The OpenZFS event daemon monitors pools. This patch enables the email sending function by default (if zed is installed). This is consistent with the default behavior of mdadm.

See: https://salsa.debian.org/zfsonlinux-team/zfs/-/blob/0b542b085ec0a4070392bd01921cfaa7ce1bad87/debian/patches/1005-enable-zed.patch
---
 cmd/zed/zed.d/zed.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmd/zed/zed.d/zed.rc b/cmd/zed/zed.d/zed.rc
index bc269b155..e6d4b1703 100644
--- a/cmd/zed/zed.d/zed.rc
+++ b/cmd/zed/zed.d/zed.rc
@@ -41,7 +41,7 @@ ZED_EMAIL_ADDR="root"
 ##
 # Minimum number of seconds between notifications for a similar event.
 #
-#ZED_NOTIFY_INTERVAL_SECS=3600
+ZED_NOTIFY_INTERVAL_SECS=3600
 
 ##
 # Notification verbosity.
-- 
2.39.2

